src/box/memtx_engine.c

```diff
@@ -216,7 +229,7 @@ memtx_engine_recover_raft(const struct xrow_header *row)
 
 static int
 memtx_engine_recover_snapshot_row(struct memtx_engine *memtx,
-				  struct xrow_header *row)
+				  struct xrow_header *row, bool *not_system_space)
```

Comment:
By out code style bool variables should start with `is_`, `was_` etc.

Reply:
Fix